filebeat/filebeat.template-es2x.json

```diff
@@ -535,6 +535,7 @@
   },
   "order": 0,
   "settings": {
+    "index.mapping.total_fields.limit": 10000,
```

Comment:
Will only apply it if not 2x as we have the logic for that. We should also make use of it :-)